container/debian8/layers/bazel/BUILD

```diff
@@ -0,0 +1,97 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:public"])
+
+load("//container/rules:docker_toolchains.bzl", "language_tool_layer")
+load("//container/rules:docker_toolchains.bzl", "toolchain_container")
+load("@base_images_docker//util:run.bzl", "container_run_and_extract")
+load(
+    "//container/common/bazel:version.bzl",
+    "BAZEL_VERSION_SHA256S",
+    "BAZEL_VERSION_TO_CONFIG_VERSION",
+)
+
+exports_files(glob(["*.yaml"]))
+
+# Used in substitution in bazel.yaml.tpl and bazel.bazelrc.tpl
+CONFIG_BASE = "debian8_clang/0.3.0"
```

Comment:
Can we put this CONFIG_BASE in a bzl file further up in the directory structure (maybe in https://github.com/bazelbuild/bazel-toolchains/blob/master/container/debian8/BUILD or in https://github.com/bazelbuild/bazel-toolchains/blob/master/configs/debian8_clang/BUILD)

Reply:
Done.